### PR TITLE
[newrelic-logging] Update Docs: "fluentBit.k8sLoggingExclude" needs to be 'true' and not 'On'

### PR DESCRIPTION
**Pull request by nr-rkallempudi**
…lse from on/off

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

**Comment by nr-rkallempudi** [comment]
**Test cases:**
Tested scenarios with following values and observed results in compiled yaml file:
fluentBit.k8sLoggingExclude: off -> false
fluentBit.k8sLoggingExclude: on -> true
fluentBit.k8sLoggingExclude: true -> true
fluentBit.k8sLoggingExclude: false -> false
Not having the attribute fluentBit.k8sLoggingExclude -> false

File: charts/newrelic-logging/README.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ See [values.yaml](values.yaml) for the default values
...
-| `fluentBit.k8sLoggingExclude`                                | Set to "On" to allow excluding pods by adding the annotation `fluentbit.io/exclude: "true"` to pods you wish to exclude.                                                                                                                                                                                                                                                             | `Off`                                                                           |
+| `fluentBit.k8sLoggingExclude`                                | Set to "True" to allow excluding pods by adding the annotation `fluentbit.io/exclude: "true"` to pods you wish to exclude.                                                                                                                                                                                                                                                             | `false`                                                                           |
```
Comment (danybmx) [inline charts/newrelic-logging/README.md:152]: Hi! "True" or "true"?

File: charts/newrelic-logging/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart to deploy New Relic Kubernetes Logging as a DaemonSet, supporting both Linux and Windows nodes and containers
 name: newrelic-logging
-version: 1.22.1
+version: 1.22.2
```
Comment (danybmx) [inline charts/newrelic-logging/Chart.yaml:4]: It should be 1.22.3

**Review by danybmx — CHANGES_REQUESTED** [review]
Amazing, just a few comments.
Would you mind to add some unit tests for this?
Regards! :bow:

**Pull request closed without merging**